front/pages/api/poke/workspaces/[wId]/data_sources/[name]/config.ts

```diff
@@ -45,7 +45,11 @@ async function handler(
           },
         });
       }
-      if (!req.body || typeof req.body.configValue !== "string") {
+      if (
```

Comment:
Given that you updated, it should be `string` only, right?

Reply:
Yes, omission too